src/Modules/Windows/Microsoft.PowerShell.Utility/Microsoft.PowerShell.Utility.psd1

```diff
@@ -37,10 +37,6 @@ PrivateData = @{
         Name        = 'Microsoft.PowerShell.Utility.PSManageBreakpointsInRunspace'
         Description = 'Enables -BreakAll parameter on Debug-Runspace and Debug-Job cmdlets to allow users to decide if they want PowerShell to break immediately in the current location when they attach a debugger.'
       }
-      @{
-        Name        = 'Microsoft.PowerShell.Utility.PSGetError'
-        Description = 'Enable Get-Error cmdlet that displays detailed information about ErrorRecords included nested objects'
```

Comment:
Why this section in psd1 was necessary?

Reply:
I believe this probably should have been only in the psd1 and not in the internal list of experimental features

Reply:
Yes, the internal list of experimental features are supposed to be engine features, and the `Source` would be `PSEngine`. For experimental features exposed from a module, it should be declared in the psd1 file, not in the engine feature list.